templates/docker-init.j2

```diff
@@ -1,12 +1,61 @@
 # Generated by Ansible for {{ansible_fqdn}}
 description "Docker daemon"
 
-start on filesystem
+start on (filesystem and net-device-up IFACE!=lo)
 stop on runlevel [!2345]
+limit nofile 524288 1048576
```

Comment:
This configuration sometimes seems to be needed  as far as I see in https://github.com/docker/docker/issues/5939 and https://github.com/docker/docker/pull/4455 (systemd), corresponding fix for upstart is https://github.com/docker/docker/pull/4749 , https://github.com/docker/docker/commit/119839dfd7b8b77eb22ccd45116f45a4bc17a1a6